src/pal/inc/pal.h

```diff
@@ -442,6 +442,16 @@ BOOL
 PALAPI
 PAL_NotifyRuntimeStarted(VOID);
 
+PALIMPORT
+LPCSTR
+PALAPI
+PAL_GetApplicationGroupId();
```

Comment:
Please do not add functions that are not used by the runtime and thus never tested.

Reply:
This was getting it ready for the change to enable debugging.  Should I remove it and add as part of that change ?

Reply:
Ah, ok, in that case it makes sense to keep them. I didn't know you had follow up plans.